src/jit/hwintrinsicxarch.cpp

```diff
@@ -153,6 +189,11 @@ InstructionSet HWIntrinsicInfo::lookupIsa(const char* className)
     {
         return InstructionSet_LZCNT;
     }
+    else if (strcmp(className, "X64") == 0)
```

Comment:
Could we shortcut this with `if (enclosingClassName == nullptr)`?

Reply:
`enclosingClassName` is a empty string, not `nullptr `.

Reply:
Ah, we can, let me change.